src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,71 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE_OF_STORAGE = 10;
+    private K[] tableOfKey;
+    private V[] tableOfValue;
+    private int arrayIndex = 0;
```

Comment:
```suggestion
    private int arrayIndex;
```

it will get default value of 0
if you still want to initialise it, it is better be done in constructor